sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/throughputControl/config/ThroughputControlGroupInternal.java

```diff
@@ -114,11 +129,21 @@ public boolean equals(Object other) {
 
         ThroughputControlGroupInternal that = (ThroughputControlGroupInternal) other;
 
-        return StringUtils.equals(this.id, that.id);
+        return StringUtils.equals(this.id, that.id)
```

Comment:
Instead of using StringUtils from apache, we should use `Objects.equals(id, that.id)` - which is coming from java.util package - which is also the default implementation of equals method.

Reply:
will update in next iteration.